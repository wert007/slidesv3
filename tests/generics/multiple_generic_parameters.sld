struct Foo<Bar, Baz> {
    field: $Bar;
    field2: $Baz;

    func $toString() -> string {
        return this.field + ' - ' + this.field2;
    }
}

func main() {
    let foo = new Foo(-10, false);
    print(foo);
    let ha = new Foo('Hello', 'Hallo');
    print(ha);
}