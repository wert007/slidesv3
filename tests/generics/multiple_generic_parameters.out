-10 - false
Hello - Hallo
= 0
